src/amo/components/LanguagePicker.scss

```diff
@@ -7,6 +7,8 @@ $selector-padding: 10px;
 }
 
 .LanguagePicker-selector {
+  -moz-appearance: none;
```

Comment:
We only [configure the auto-prefixer](https://github.com/mozilla/addons-frontend/blob/master/webpack.prod.config.babel.js#L91) for prod builds so you can test it there. Maybe our 'last 2 versions' setting needs to be tweaked?